Lombiq.VueJs.Samples/Controllers/QrCardController.cs

```diff
@@ -0,0 +1,41 @@
+using Lombiq.VueJs.Samples.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.ContentManagement;
+using OrchardCore.Mvc.Core.Utilities;
+using System.Threading.Tasks;
+
+namespace Lombiq.VueJs.Samples.Controllers;
+
+public class QrCardController : Controller
+{
+    private readonly IContentManager _contentManager;
+
+    public QrCardController(IContentManager contentManager) =>
+        _contentManager = contentManager;
+
+    // Open this from under /Lombiq.VueJs.Samples/QrCard/Index.
```

Comment:
Make this an XmlDoc comment, so it can be seen on hover where it's used, too:
```suggestion
    /// <remarks><para>Open this from under /Lombiq.VueJs.Samples/QrCard/Index.</para></remarks>
```